galasa-ui/src/app/mysettings/page.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+'use client'
+
+import { useEffect, useState } from "react";
+import { Loading, Button} from "@carbon/react"
+import PageTile from "@/components/PageTile";
+import styles from "../../styles/MySettings.module.css"
+import TokenCard from "@/components/TokenCard";
+import ErrorPage from "../error/page";
+import TokenRequestModal from "@/components/TokenRequestModal";
+import TokenDeleteModal from "@/components/TokenDeleteModal";
+import TokenResponseModal from "@/components/TokenResponseModal";
+
+export default function MySettingsPage() {
+
+    const [isLoading, setIsLoading] = useState(false)
+    const [isError, setIsError] = useState(false)
+    const [selectedTokens, setSelectedTokens] = useState<Set<string>>(new Set());
+    const [tokens, setTokens] = useState<Set<Token>>(new Set())
+    const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false)
+
+    interface Token {
+        tokenId: string;
+        description: string;
+        creationTime: string;
+        owner: {
+            loginId: string;
+        };
+    }
+
+    const handleSelectTokensForDeletion = (tokenId: any) => {
+
+        if (selectedTokens.has(tokenId)) {
+            selectedTokens.delete(tokenId);
+
+            setSelectedTokens((prevSelectedTokens) => {
+                const newSet = new Set(prevSelectedTokens);
+                newSet.delete(tokenId);
+                return newSet;
+            });
+        }
+        else {
+            setSelectedTokens((prevSelectedTokens) => {
+                const newSet = new Set(prevSelectedTokens);
+                newSet.add(tokenId);
+                return newSet;
+            });
+
+        }
+    }
+
+    const handleFetchAccessTokens = async () => {
```

Comment:
Minor - we don't need to have "handle" prefixed to a lot of functions. In this case, `fetchAccessTokens` would be clear enough. Same applies throughout this PR.